extensions/amp-story/1.0/test/test-amp-story-bookend.js

```diff
@@ -30,273 +30,359 @@ import {createElementWithAttributes} from '../../../../src/dom';
 import {registerServiceBuilder} from '../../../../src/service';
 import {user} from '../../../../src/log';
 
-describes.realWin('amp-story-bookend', {amp: true}, env => {
-  let win;
-  let doc;
-  let storyElem;
-  let bookend;
-  let bookendElem;
-  let requestService;
-  let analytics;
-  let analyticsVariables;
-
-  const expectedComponents = [
-    {
-      'type': 'heading',
-      'text': 'My Heading Title!',
+describes.fakeWin(
+  'amp-story-bookend',
+  {
+    win: {
+      location:
```

Comment:
Can we do something like this instead? It's almost impossible to review this change otherwise :(
```
const location = 'foo';
describes.realWin('amp-story-bookend', {location, amp: true}, env => {
```

Reply:
Sure, just note that we have to use `fakeWin` in order to use `win.location.resetHref`